packages/cursorless-engine/typings/Graph.ts

```diff
@@ -0,0 +1,55 @@
+import { CommandServerApi, Range, TextDocument } from "@cursorless/common";
```

Comment:
moved graph type into its own file so that we don't need to export the whole big `Types.` file just to export the graph type